src/GeositeFramework/css/main.css

```diff
@@ -2335,3 +2335,7 @@ body.x-body {
 .launchpad-plugin li p {
   font-size: 13px;
 }
+.icon-wrench:before {
```

Comment:
I'd make this selector more specific, perhaps referencing the parent button. That way, if someone uses this icon elsewhere in the application, it won't be impacted by this style that is made for centering on a button.